cmd/server/main.go

```
@@ -29,10 +153,163 @@ func main() {
+	// cmd.Dir = config.WorkingDir
+
+	cmd.Stdout = logFile
+	cmd.Stderr = logFile
+
+	if err := cmd.Start(); err != nil {
+		logging.LogAction("START_FAILED", fmt.Sprintf("Failed to start daemon: %v", err))
+		return // Just exit the function
+	}
+
+	if err := writePID(cmd.Process.Pid); err != nil {
+		cmd.Process.Kill()
```

golangci-lint

Error return value of `cmd.Process.Kill` is not checked

(errcheck)